happi/backends/qs_db.py

```diff
@@ -244,16 +244,29 @@ def _create_db_item(info: dict,
         # 1. A capitalized name:
         name = name.lower()
 
+        pv_prefix = info['pvbase']
+
+        if class_name and class_name in (
```

Comment:
Maybe I should be refactoring things a bit here to handle the code for creating entries separately for every device or something of this sort?
It could get messy here very soon the way I am doing it now...
I am also working on getting the `mpod` here as well, and I'll have to handle some information specific for `mpod`  too, as it has instead of `pvbase` -> `pvname` and I also have to send custom `kwargs` ..

Any suggestion here would be appreciated :)